apps/src/templates/rubrics/RubricContent.jsx

```diff
@@ -119,50 +119,56 @@ export default function RubricContent({
 
   return (
     <div className={style.rubricContent}>
-      {!!studentLevelInfo && <Heading2>{studentLevelInfo.name}</Heading2>}
-      <Heading5>
-        {i18n.lessonNumbered({
-          lessonNumber: lesson.position,
-          lessonName: lesson.name,
-        })}
-      </Heading5>
-      {!!studentLevelInfo && (
-        <div className={style.studentInfo}>
-          <div className={style.levelAndStudentDetails}>
-            {onLevelForEvaluation && (
-              <div className={style.studentMetadata}>
-                {studentLevelInfo.timeSpent && (
+      <div>
+        {!!studentLevelInfo && (
+          <Heading2 style={{marginBottom: '8px'}}>
```

Comment:
I could not figure out a way to override the `Heading2` styles so I had t do this inline... would love feedback on this approach though!